Tests/X509Tests/PolicyBuilderTests.swift

```diff
@@ -328,4 +408,153 @@ final class PolicyBuilderTests: XCTestCase {
             }
         }
     }
+
+    func testChainFailsPolicy_OneOf_empty() async {
+        await assertFailsToMeetPolicy {
+            OneOfPolicies {}
+        }
+        let `false` = false
+        // This is effectively empty because the branch is never true
+        await assertFailsToMeetPolicy {
+            OneOfPolicies {
+                if `false` {
+                    Policy(result: .meetsPolicy)
+                }
+            }
+        }
+
+        let policy: Policy? = nil
+        // This is effectively empty because the optional is always nil
+        await assertFailsToMeetPolicy {
+            OneOfPolicies {
+                if let policy {
+                    policy
+                }
+            }
+        }
```

Comment:
Nit: we are missing a test case for a non-nil `WrappedOptional` that meets/fails the policy.